config

```diff
@@ -57,27 +57,34 @@ else
     fi
 fi
 
-if [ $ngx_found = yes ]; then
-    CORE_INCS="$CORE_INCS $ngx_feature_path"
-    CORE_LIBS="$CORE_LIBS $ngx_feature_libs"
-else
+if [ $ngx_found = no ]; then
  cat << END
  $0: error: ngx_http_replace_filter_module requires agentzh's sregex library.
 END
  exit 1
 fi
 
-ngx_addon_name=ngx_http_replace_filter_module
-HTTP_AUX_FILTER_MODULES="$HTTP_AUX_FILTER_MODULES ngx_http_replace_filter_module"
-
-NGX_ADDON_SRCS="$NGX_ADDON_SRCS \
-                $ngx_addon_dir/src/ngx_http_replace_filter_module.c \
-                $ngx_addon_dir/src/ngx_http_replace_script.c \
-                $ngx_addon_dir/src/ngx_http_replace_parse.c \
-                $ngx_addon_dir/src/ngx_http_replace_util.c"
+_REPLACE_SRCS="$ngx_addon_dir/src/ngx_http_replace_filter_module.c \
```

Comment:
It's a bad idea to use variable names with leading underscores. Please avoid it. Maybe `REPLACE_FILTER_SRCS` is the right name for this.